src/OVAL/results/oval_resultTest.c

```diff
@@ -465,6 +465,132 @@ static inline oval_result_t _evaluate_sysent(struct oval_syschar_model *syschar_
 	}
 }
 
+struct oval_item_statuses {
```

Comment:
Struct in the middle of the file? I would look for it at the beginning but it's just my opinion.